dspot/src/main/java/fr/inria/stamp/Main.java

```diff
@@ -17,104 +17,141 @@
+		}
+	}
 
-    static void runExample() {
-        try {
-            InputConfiguration configuration = new InputConfiguration(
-                    "src/test/resources/test-projects/test-projects.properties");
-            DSpot dSpot = new DSpot(configuration, 1, Collections.singletonList(new TestDataMutator()),
-                    new JacocoCoverageSelector());
-            dSpot.amplifyTest("example.TestSuiteExample");
-        } catch (Exception e) {
-            throw new RuntimeException(e);
-        }
-    }
+	static void runExample() {
+		try {
+			InputConfiguration configuration = new InputConfiguration(
+					"src/test/resources/test-projects/test-projects.properties");
+			DSpot dSpot = new DSpot(configuration, 1, Collections.singletonList(new TestDataMutator()),
+					new JacocoCoverageSelector());
+			dSpot.amplifyTest("example.TestSuiteExample");
+		} catch (Exception e) {
+			throw new RuntimeException(e);
+		}
+	}
+
+	private static void print(Configuration configuration, InputConfiguration inputConfiguration) {
+		LOGGER.info("==== CONFIGURATION ====");
+		LOGGER.info("path to configuration file: " + configuration.pathToConfigurationFile);
+		LOGGER.info("amplifiers: " + configuration.amplifiers);
+		LOGGER.info("nb iteration: " + configuration.nbIteration);
+		LOGGER.info("test classes: " + configuration.testClasses);
+		LOGGER.info("path to output: " + configuration.pathToOutput);
+		LOGGER.info("selector: " + configuration.selector);
+		LOGGER.info("test cases: " + configuration.testCases);
+		LOGGER.info("seed: " + configuration.seed);
+		LOGGER.info("time out in ms: " + configuration.timeOutInMs);
+		LOGGER.info("automatic builder name: " + configuration.automaticBuilderName);
+		LOGGER.info("maven home: " + configuration.mavenHome);
+		LOGGER.info("max test amplified: " + configuration.maxTestAmplified);
+		LOGGER.info("clean: " + configuration.clean);
+		LOGGER.info("minimize: " + configuration.minimize);
+		LOGGER.info("==== INPUTCONFIGURATION ====");
+		Properties prop = inputConfiguration.getProperties();
+		Set keys = prop.keySet();
+		for (Object key : keys) {
+			LOGGER.info(key + ": " + prop.getProperty((String) key));
+		}
+		LOGGER.info("ClassesDir: " + inputConfiguration.getClassesDir());
+		LOGGER.info("coverageDir: " + inputConfiguration.getCoverageDir());
+		LOGGER.info("outputDirectory: " + inputConfiguration.getOutputDirectory());
+		LOGGER.info("previousTransformationPath: " + inputConfiguration.getPreviousTransformationPath());
+		LOGGER.info("projectPath: " + inputConfiguration.getProjectPath());
+		LOGGER.info("relativeSourceCodeDir: " + inputConfiguration.getRelativeSourceCodeDir());
+		LOGGER.info("relativeTestSourceCodeDir: " + inputConfiguration.getRelativeTestSourceCodeDir());
+		LOGGER.info("RootPath: " + inputConfiguration.getRootPath());
+		LOGGER.info("TempDir: " + inputConfiguration.getTempDir());
+		LOGGER.info("ValidationErrors: " + inputConfiguration.getValidationErrors());
+	}
```

Comment:
I kind of disagree with this large print... Quite useful for debugging I agree, but It appears to me as spam for users. Anyway, I would rather put this display code in a `toString()` methods of both objects `Configuration` and `InputConfiguration`, what do you think?

Reply:
Yes it's for debug issue. It was useful for me to compare the parameter passed from dspot.propertis file with the parameter passed from pom.xml.
I agree we can move on toString() and activate it only on verbose mode.